dts/arm/nxp/nxp_lpc54xxx.dtsi

```diff
@@ -12,8 +12,8 @@
 		gpio-0 = &gpio0;
 		gpio-1 = &gpio1;
 		mailbox-0 = &mailbox0;
-		spi-5 = &spi5;
-		usart-0 = &usart0;
+		spi-5 = &flexcomm5;
```

Comment:
are the flexcomm's a fix function?  If not, these alias setting might be better in the board file.  (same question for the LPC55S69 aliases).

Reply:
You're right, good catch. But aren't these instance aliases going away soon anyway?

Reply:
They are, so you could also just remove these for now.

Reply:
I can't remove them until the drivers are converted, but I did move them to the board file as suggested.